{"errors":["In the world's resolve_all_references step: In 'invalid_times_played': Invalid argument: Round-robin argument 'num_times_to_play_each_opponent' was 0. Must be 1 or more, or not set."]}